### ISSUE TITLE
Sequel::SQL::Function is deprecated

### ISSUE THREAD

**Issue by mvasin** [jeremyevans/roda-sequel-stack#8]
I'd like to fix the deprecation warning (that I also have in my app, derived from this repo), but I'm not sure how.

```
iMac-5K:roda-sequel-stack mvasin$ rake
/Users/mvasin/.rbenv/versions/2.4.1/bin/ruby -e 'ARGV.each{|f| require f}' ./spec/model/*_spec.rb
SEQUEL DEPRECATION WARNING: Passing a block to a virtual row method to create a Sequel::SQL::Function is deprecated and will be removed in Sequel 5.  Replace the block with a call to .function to create a function, or use the virtual_row_method_block extension.
/Users/mvasin/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/sequel-4.46.0/lib/sequel/sql.rb:1949:in `method_missing'
/Users/mvasin/tmp/roda-sequel-stack/spec/model/spec_helper.rb:3:in `block in <top (required)>'
/Users/mvasin/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/sequel-4.46.0/lib/sequel/core.rb:413:in `instance_exec'
/Users/mvasin/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/sequel-4.46.0/lib/sequel/core.rb:413:in `virtual_row'
/Users/mvasin/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/sequel-4.46.0/lib/sequel/dataset/query.rb:1370:in `virtual_row_columns'
/Users/mvasin/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/sequel-4.46.0/lib/sequel/dataset/query.rb:821:in `select'
/Users/mvasin/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/sequel-4.46.0/lib/sequel/dataset/actions.rb:279:in `get'
/Users/mvasin/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/sequel-4.46.0/lib/sequel/database/query.rb:70:in `get'
/Users/mvasin/tmp/roda-sequel-stack/spec/model/spec_helper.rb:3:in `<top (required)>'
/Users/mvasin/tmp/roda-sequel-stack/spec/model/model1_spec.rb:1:in `require_relative'
Run options: --seed 53502

# Running:

.

Finished in 0.001107s, 903.3424 runs/s, 0.0000 assertions/s.

1 runs, 0 assertions, 0 failures, 0 errors, 0 skips
/Users/mvasin/.rbenv/versions/2.4.1/bin/ruby -e 'ARGV.each{|f| require f}' ./spec/web/*_spec.rb
SEQUEL DEPRECATION WARNING: Passing a block to a virtual row method to create a Sequel::SQL::Function is deprecated and will be removed in Sequel 5.  Replace the block with a call to .function to create a function, or use the virtual_row_method_block extension.
/Users/mvasin/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/sequel-4.46.0/lib/sequel/sql.rb:1949:in `method_missing'
/Users/mvasin/tmp/roda-sequel-stack/spec/web/spec_helper.rb:3:in `block in <top (required)>'
/Users/mvasin/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/sequel-4.46.0/lib/sequel/core.rb:413:in `instance_exec'
/Users/mvasin/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/sequel-4.46.0/lib/sequel/core.rb:413:in `virtual_row'
/Users/mvasin/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/sequel-4.46.0/lib/sequel/dataset/query.rb:1370:in `virtual_row_columns'
/Users/mvasin/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/sequel-4.46.0/lib/sequel/dataset/query.rb:821:in `select'
/Users/mvasin/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/sequel-4.46.0/lib/sequel/dataset/actions.rb:279:in `get'
/Users/mvasin/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/sequel-4.46.0/lib/sequel/database/query.rb:70:in `get'
/Users/mvasin/tmp/roda-sequel-stack/spec/web/spec_helper.rb:3:in `<top (required)>'
/Users/mvasin/tmp/roda-sequel-stack/spec/web/prefix1_spec.rb:1:in `require_relative'
Run options: --seed 18626

# Running:

.

Finished in 0.001528s, 654.4503 runs/s, 0.0000 assertions/s.

1 runs, 0 assertions, 0 failures, 0 errors, 0 skips
```

**Comment by jeremyevans** [comment]
Thanks for the heads up. This issue should definitely be fixed.  I'll commit a patch shortly.

**Comment by jeremyevans** [comment]
Fixed by 90b9cb415caa8040c71e3d647e4e5471acf5b0e9
